### PR TITLE
Administrator interface redesign

### PR DESCRIPTION
**Pull request by jlyon1**
**Changes:**
-no polymer
-no three tab layout
+allow adding routes
+implement live route snapping when creating new routes
+tie stops directly to routes
+force stop to have an associated route
+allow adding stops
+allow editing stops

### CODE REVIEW

**Comment by jlyon1** [comment]
fixes #74

File: api/routes.go [hunk 1]
```diff
@@ -136,151 +53,9 @@ func ComputeDistanceMapPoint(c1 model.MapPoint, c2 model.MapPoint) float64 {
...
-	fmt.Printf(name)
-	coords = []model.Coord{}
-	err = App.db.Routes.Insert(&newRoute)
-
-}
-
 // RoutesCreateHandler adds a new route to the database
 func (App *API) RoutesCreateHandler(w http.ResponseWriter, r *http.Request) {
-	// Create a new route object using request fields
+	// Create a new route object using reques`t fields
```
Comment (kochman) [inline api/routes.go]: lol

File: api/routes.go [hunk 2]
```diff
@@ -304,11 +79,11 @@ func (App *API) RoutesCreateHandler(w http.ResponseWriter, r *http.Request) {
 		coord := model.Coord{c["lat"], c["lng"]}
 		coords = append(coords, coord)
 	}
+
 	// Here do the interpolation
-	coords = Interpolate(coords, App.cfg.GoogleMapAPIKey)
-	segments := ComputeSegments(coords, App.cfg.GoogleMapAPIKey, App.cfg.GoogleMapMinDistance)
 	// now we get the Segment for each segment ( this should be stored in database, just store it inside route for god sake)
 
+	//segments := ComputeSegments(coords, App.cfg.GoogleMapAPIKey, App.cfg.GoogleMapMinDistance)
```
Comment (kochman) [inline api/routes.go]: nuke?

**Comment by jlyon1** [comment]
fixes #61

**Pull request merged**